services/src/api/validators/campaigns.val.ts

```diff
@@ -0,0 +1,77 @@
+import { Schema } from 'express-validator';
+
+export const CampaignCreate: Schema = {
+    name: {
+        in: ['body'],
+        isString: { errorMessage: 'Name must be of type string' },
+        exists: { errorMessage: 'Name must be defined' },
+        trim: true,
+        escape: true,
+    },
+    startAt: {
+        in: ['body'],
+        isNumeric: { errorMessage: 'Start At must be of type number(date)' },
```

Comment:
Should be used with isInt, with 0 as min option

See https://www.npmjs.com/package/validator for all options